skt/__init__.py

```diff
@@ -12,19 +12,20 @@
 # along with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 
-from email.errors import HeaderParseError
 import email.header
 import email.parser
+import io
 import logging
 import multiprocessing
 import os
 import re
 import shutil
 import subprocess
 import sys
-import io
 import time
+from email.errors import HeaderParseError
```

Comment:
Moving this introduces `pylint` error `C: 26, 0: Imports from package email are not grouped (ungrouped-imports)`.